src/app/components/user/cognito-user-edit.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
+import {
+  arrayOf,
+  func,
+  string,
+} from 'prop-types';
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  FormControlLabel,
+  Switch,
+  ThemeProvider,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import theme from '../../themes/theme';
+import { ChplTextField } from '../util';
+import { ChplActionBar } from '../action-bar';
+import {
+  user as userPropType,
+} from '../../shared/prop-types';
```

Comment:
```suggestion
import theme from 'themes/theme';
import { ChplTextField } from 'components/util';
import { ChplActionBar } from 'components/action-bar';
import {
  user as userPropType,
} from 'shared/prop-types';
```

We don't do relative imports, except for "local relative". I usually sort imports by the values after the `from` keyword as well